### ISSUE TITLE
Problem with remap

### ISSUE THREAD

**Issue by claychinasky** [kprinssu/anne-keyboard-windows#32]
I can't seem  to remap FN key to any key. It simply doesn't work as rightside FN key does. The key seems to mapped as blank I think.

**Comment by ethanmsmith** [comment]
Did you un-map the right fn key? The keyboard firmware only allows one fn key

**Comment by claychinasky** [comment]
Yes, I did. FN + 1 keys are working as F keys but WASD and other keys are not working with FN key remmaped on LWIN key. When I try FN+WASD it snaps windows to screen and when I press only FN key it pops up the windows start menu. I guess its firmware issue then ?

**Comment by ethanmsmith** [comment]
Sounds like it's both fn and win key... I'll take a look into it.

**Comment by claychinasky** [comment]
Ok figured out. Both function layer and base layer mapping needs to match. What was happening once FN key is pressed it turns itself to LWIN key. After I correct it to FN key, it worked as expected.

**Comment by ethanmsmith** [comment]
I think this is working as intended tbh. Close this?

**Comment by kprinssu** [comment]
This part of the app was hacked together. The reason why you need to have the FN key in both the base and function layer is to ensure that you can actually toggle back between the two layers.

 Proper documentation should be added.
